cmd/copy.go

```diff
@@ -433,11 +435,21 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 		glcm.SetOutputFormat(common.EOutputFormat.None())
 	}
 
-	cooked.preserveNTFSACLs = raw.preserveNTFSACLs
-	if err = validatePreserveNTFSACLs(cooked.preserveNTFSACLs, cooked.fromTo); err != nil {
+	cooked.preserveSMBPermissions = raw.preserveSMBPermissions
+	if err = validatePreserveSMBPropertyOption(cooked.preserveSMBPermissions, cooked.fromTo, "preserve-smb-permissions"); err != nil {
 		return cooked, err
 	}
 
+	cooked.preserveSMBProperties = raw.preserveSMBProperties
+	if err = validatePreserveSMBPropertyOption(cooked.preserveSMBProperties, cooked.fromTo, "preserve-smb-properties"); err != nil {
+		return cooked, err
+	}
+
+	// Since we already don't support preserving LMT on upload or S2S, it's fine to limit this to download.
+	if cooked.fromTo.IsDownload() && cooked.preserveSMBProperties && cooked.preserveLastModifiedTime {
+		return cooked, errors.New("SMB properties include a last modified time. Please use either --preserve-smb-properties OR --preserve-last-modified-time")
```

Comment:
I think I've just realised this is incorrect.  Why? Because sync also sets preserveLastModifiedTime in the job order - and its hard coded to always set it. So we can reject it being set, in the case of sync.

So, I think we probably need to change the code in the STE where we preserve the LMT, to say that we only preserve it if we are NOT preserving SMB properties.  I.e. let user set both, but only do the preservation once?  Thoughts?

Reply:
I think I agree with preferring the SMB property if explicitly specified.